lib/storage/providers/LocalForage.js

```diff
@@ -57,12 +62,37 @@ const provider = {
      * @return {Promise<void>}
      */
     multiMerge(pairs) {
-        const tasks = _.map(pairs, ([key, value]) => this.setItemQueue.push({key, value, shouldMerge: true}));
+        const tasks = _.map(pairs, ([key, value, currentValue]) => {
```

Comment:
If the parameter is a no longer a key/value pair but also contains `currentValue`, the variable should no longer be called `pairs` 

---

Have we searched for ways to not pass the `currentValue` for each key?

Since this is being queued it's possible that `currentValue` would no longer be current at the time of the write

This is the only storage provider that needs `currentValue` to work - the rest don't have such parameter